test/test_device_run_length_encode.cu

```diff
@@ -779,11 +779,11 @@ void TestSize(
         unsigned int max_int = (unsigned int) -1;
         for (int i = 0; i < 10; ++i)
         {
-            unsigned int num_items;
-            RandomBits(num_items);
-            num_items = (unsigned int) ((double(num_items) * double(10000000)) / double(max_int));
-            num_items = CUB_MAX(1, num_items);
-            TestDispatch<T, OffsetT, LengthT>(num_items);
+            unsigned int num;
+            RandomBits(num);
+            num = (unsigned int) ((double(num) * double(10000000)) / double(max_int));
```

Comment:
Since we're changing this already, maybe `static_cast` here instead?

Reply:
Done.